Configuration/SiteConfiguration/Overrides/sites.php

```diff
@@ -17,6 +17,16 @@
         'itemsProcFunc' => \B13\Bolt\Configuration\PackageHelper::class . '->getSiteListForSiteModule',
     ],
 ];
+
+$GLOBALS['SiteConfiguration']['site']['columns']['typoScriptTemplatePath'] = [
+    'label' => 'Custom path to the TypoScript template',
+    'description' => '[EXT:bolt] Defined a custom path to the TypoScript template to be included instead of the default "Configuration/TypoScript/".',
```

Comment:
[EXT:bolt] Path to the TypoScript and TsConfig entry points inside the site package (defaults to "Configuration/TypoScript/")

Reply:
Thanks, will have to look into this, because i did not modify the `TsConfig/Loader.php`.
IMO, this should be done as well.